admin-ui/app/views/notification.html

```diff
@@ -54,7 +54,7 @@
                     </tr>
                     <tr>
                       <td>Message:</td>
-                      <td colspan="2"><strong>{ "message" : "{{ activityCtrl.parse(metric).message.alert }}" }</strong></td>
+                      <td colspan="2"><strong>{ "message" : "{{ activityCtrl.parse(metric).alert }}" }</strong></td>
```

Comment:
I wonder whether it make sense to wrap the alert in `{ "message" : "...alert..."}` when it is not actually the message format (in fact it is not used anywhere in this form afaik).

It appears in the notification detail as:

````
Request IP:	127.0.0.1	Full Request
Message:	{ "message" : "test" }
````

Could we state just: `Message: {{ activityCtrl.parse(metric).alert }}`